dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -207,4 +207,15 @@
             DROP FUNCTION check_workflow_checker_id_is_unique;
         </sql>
     </changeSet>
+    <changeSet author="svonworl" id="addVerifiedPlatformsToVersionMetadata">
+        <addColumn tableName="version_metadata">
+            <column name="verifiedplatforms" type="VARCHAR(255)"/>
+        </addColumn>
+        <sql dbms="postgresql">
+            UPDATE version_metadata vm
+            SET verifiedPlatforms = '["' || sv.source || '"]'
```

Comment:
I don't know our data model very well here and my SQL is weak as well, so I don't entirely understand this statement.

Couldn't there be more than one platform for a given version? Would this code handle it if so?

I poked around a little and I think the model allows it, but we don't actually have such data. If that's the case, I think it's fine, as this data won't be changing before the release. It could be worth a code comment.

If I'm totally off-base, please clarify. Thanks!

Reply:
Yup, a version can have multiple different verified platforms, but there aren't any such versions in our database right now.  In the case there are multiple platforms, it picks one.  Yeah, it's a bit of a hack, and if anyone objects, I can make it do better, but IMHO it's not worth the time.  In lieu of that, I will add a code comment. :)

Reply:
I think that's fine since we may steer users away from this style of verification toward metric's style verification anyway